### PR TITLE
Toolbar & headline section

### PR DESCRIPTION
**Pull request by CarlosE-Feliz**
### Toolbar and Headline section 😬

>Open pull request to check for issues😴

>Waiting for approbation to pull my codes to my main branch😎😎😎😎

> Waiting to keep working on my portfolio

### CODE REVIEW

**Comment by CarlosE-Feliz** [comment]
### Hey there!😁

After several attempts🙄, all errors have been corrected🥳🥳🥳.

Can you check this?
can I merge my pull request?
------

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments

File: index.html [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./css/styles.css">
+    <title>My portfolio</title>
+</head>
+<body>
+    <div class="header">
+        <nav class="nav-bar">
```
Comment (arslanbisharat) [inline index.html:12]: -  [x] Please make sure you are using header tag.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+            <div class="name">
+                <h4 class="carlosfeliz">Carlos Feliz</h4>
```
Comment (arslanbisharat) [inline index.html]: Lolo should be a link and encloswd in a tag

File: index.html [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+                    </a>
+
+                 <a href="#" class="anchor"> 
+                     <h4 class="contact"> Contact </h4>
+                     </a>
+            </div>
+             
+        </nav>
+    </div>
+    <header class="main">
```
Comment (arslanbisharat) [inline index.html]: Instead of header please use section tag

File: index.html [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+            <div class="frame2">
+                <div class="language">
+                    <div class="header-language">
+                        <h3 class="text-language">Language</h3>
+                    </div>
+                    <div class="main-language">
+                        <p class="p-css">CSS</p>
+                        <p class="p-html">HTML</p>
+                        <p class="p-javascript">JavaScript</p>
+                    </div>
```
Comment (arslanbisharat) [inline index.html:59]: You should use ul and li tags

File: index.html [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+                
+                    <div class="frameworks">
+                        <div class="header-frameworks">
+                            <h3 class="text-frameworks">Frameworks</h3>
+                        </div>
+                        <div class="main-frameworks">
+                            <p class="p-framework1">N/A</p>
+                            <p class="p-framework2">N/A</p>
+                            <p class="p-framework3">N/A</p>
+                        </div>
```
Comment (arslanbisharat) [inline index.html:75]: You should use ul and li tags

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
# Changes Requested, 

Hi, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: index.html (hunk 4 above)
Comment (CarlosE-Feliz) [reply index.html:59]: They should be inside an `<a>`?

Comment (CarlosE-Feliz) [reply index.html:59]: Or I'll add that after this time?

File: index.html (hunk 2 above)
Comment (CarlosE-Feliz) [reply index.html]: Like a random one?? cause I don't have a logo.
What should I do in this case?

**Comment by CarlosE-Feliz** [comment]
##Update tags😎

- Add header instead of div.😉
- Add ul and li . (¬‿¬)
- Add section instead of header. 😅
- remove some issues and commit my changes. 🤐🤐
- Add an anchor to my logo as that way now is a link. 😬😬😬 

In case it is necessary to add a logo
--------

> while working on my portfolio, i will work on a logo with my name

> I would have to create my own logo, I don't want to use a random one. 

> I think I have fixed all the errors that I had previously been given.

**Review by voscarmv — APPROVED** [review]
Hi @xJokeer , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**